client/src/redux/reducers/twitterReducer.js

```diff
@@ -10,7 +14,6 @@ const initialState = {
   screenName: '',
   profileImage: '',
   isConnected: false,
-  home_timeline_tweets: [],
   streams: [],
```

Comment:
There's a problem with this structure: Whenever you need to modify a property from a stream, you need to create a new array of streams, which triggers a lot of unnecessary rerenders. Remind me to discuss this on Monday.
[NO ACTION REQUIRED]